Gemfile

```diff
@@ -96,6 +96,9 @@ end
 # Multitenancy
 gem 'acts_as_tenant'
 
+# Extension for validating hostnames and domain names
+gem 'validates_hostname', '~> 1.0.0'
```

Comment:
I personally rather not use the version specifier unless necessary. So I'll just put `gem 'validates_hostname'`. If it breaks because of an update, we should update our code.

Rails is probably the only exception.